File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/settings/SettingsEvent.kt
package com.hhp227.concafe.presentation.settings

sealed interface SettingsEvent {
    data object NavigateBack : SettingsEvent
    data object NavigateToAccountSettings : SettingsEvent
    data object NavigateToNotificationSettings : SettingsEvent
    data object NavigateToInquiryLink : SettingsEvent
    data class NavigateToExternalLink(val title: String, val url: String) : SettingsEvent
}

File: iosApp/iosApp/Presentation/Settings/SettingsEvent.swift
//
//  SettingsEvent.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/07.
//

import Foundation

enum SettingsEvent {
    case navigateBack
    case navigateToAccountSettings
    case navigateToNotificationSettings
    case navigateToInquiry
    case navigateToExternalLink(title: String, url: String)
}
